### PR TITLE
Retip: Use deeplinks on mobile

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

**Comment by thepiwo** [comment]
@davidyuk can you provide some instructions on how to try this out?

**Comment by davidyuk** [comment]
@thepiwo Sure, first of all, you have to install the wallet.

For iOS:
- create xcode project according to instruction in the repo
- enable signing using Aeternity team
- ensure that `applinks:mobile.z52da5wt.xyz` is in Associated domains feature
- build it to phone and try to retip something in the aggregator

For Android:
- build a production version signed by the special key
or try a prebuild version (I will sent both of them to PM)

**Comment by mradkov** [comment]
> @thepiwo Sure, first of all, you have to install the wallet (should be used `fix-callback-url-opening` branch).
> 
> For iOS:
> 
>     * create xcode project according to instruction in the repo
> 
>     * enable signing using Aeternity team
> 
>     * ensure that `applinks:mobile.z52da5wt.xyz` is in Associated domains feature
> 
>     * build it to phone and try to retip something in the aggregator
> 
> 
> For Android:
> 
>     * build a production version signed by the special key
>       or try a prebuild version (I will sent both of them to PM)

Can you provide some test cases for this?

**Comment by thepiwo** [comment]
@mradkov I already provided feedback to @davidyuk in private.

 - works with chrome on android
 - not intuitive with firefox on android

**Comment by thepiwo** [comment]
@davidyuk is there an update on this?

**Comment by davidyuk** [comment]
@thepiwo I'm not planning to change anything here, just to polish mobile.z52da5wt.xyz
can it be merged now?

**Comment by mradkov** [comment]
Also something that we did not think about - will base aepp deeplinks also work? I saw people trying to open this in the base aepp's browser and failed to tip/retip.

cc @davidyuk

**Comment by davidyuk** [comment]
There are no plans to implement deeplinks in the Base app. This PR is compatible with the usual iframe approach because I'm checking that we are not in iframe before using deeplinks.

**Comment by mradkov** [comment]
> There are no plans to implement deeplinks in the Base app. This PR is compatible with the usual iframe approach because I'm checking that we are not in iframe before using deeplinks.

Sure. Then we can merge this.

File: src/components/Retip.vue [hunk 1]
```diff
@@ -38,17 +44,24 @@
...
       Loading,
       FiatValue
     },
     computed: {
-      ...mapGetters(['settings']),
       eventPayload() {
        return `${this.tipid}:${this.retipIcon}`
+      },
+      deepLink() {
+        const url = new URL('https://mobile.z52da5wt.xyz/retip');
```
Comment (mradkov) [inline src/components/Retip.vue]: We can put the deepLink URL in a config file probably.

**Pull request merged**